### PR TITLE
IMPR/GRID-266 - First pass 

### PR DESCRIPTION
**Pull request by Dwaynekj**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ _For a list of changes since the previous version, 1.0.3 (27 May 2016), click [h
 
 ##### Developer Tutorial
 
-This [tutorial](https://openfin.github.io/fin-hypergrid) is a tool that shows developers how to use Hypergrid and implement its features.
+This [tutorial](https://openfin.github.io/fin-hypergrid/doc/tutorial-getting-starte.html) is a tool that shows developers how to use Hypergrid and implement its features.
```
Comment (RaghavPro) [inline README.md]: @Dwaynekj You missed the `d` in started*

File: src/jsdoc/tutorials/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
+## Your First HyperGrid
+ Awesome! So your ready to make your first Hypergrid.
```
Comment (RaghavPro) [inline src/jsdoc/tutorials/getting-started.md]: You're* 😆

File: README.md (hunk 1 above)
Comment (Dwaynekj) [inline README.md]: Oops! Thx

File: src/jsdoc/tutorials/getting-started.md (hunk 2 above)
Comment (Dwaynekj) [inline src/jsdoc/tutorials/getting-started.md]: Hehe. Sleepy coding

**Comment by Dwaynekj** [comment]
@joneit As discussed before merging these breaking changes. We should bump to 1.1.0 instead of 1.0.8

**Comment by joneit** [comment]
@Dwaynekj We'll need a bullet list for the readme of exactly what breaks.

**Pull request closed without merging**